docusaurus/src/pages/index.js

```diff
@@ -37,40 +43,113 @@ export default function Home() {
 
     return (
         <OpenZitiLayout>
-            <div style={{display:"flex", alignItems:"center", flexDirection:"column"}}>
-                <ThemedImage
-                    alt="OpenZiti Logo"
-                    style={{paddingTop: "75px", paddingBottom: "10px", width:"50%", alignSelf: "center", minWidth: "400px"}}
-                    sources={{
-                        light: useBaseUrl('/img/ziti-logo-dark.svg'),
-                        dark: useBaseUrl('/img/ziti-logo-light.svg'),
-                    }}
-                />
-                <div style={{display:"flex", justifyContent:"center"}}>
-                <p style={{fontSize: "x-large"}}>-- Replacing Infrastructure With Software --</p>
+            <OpenZitiHorizontalSection style={{padding: "0px"}}>
+                <div className={styles.titleSection} style={{backgroundColor: "var(--callout-color)"}}>
+                    <H2>Application Embedded Zero Trust</H2>
+
+                    <p>OpenZiti is taking application and API security to the  to making secure connectivity simple and accessible by <Highlight>replacing
+                        infrastructure with software</Highlight>. The world is software, your network security needs to
+                        react with the speed of software; it needs to be software. OpenZiti is bringing ground-breaking,
+                        zero trust networking principles directly inside applications through SDKs but OpenZiti isn't just
+                        for developers. The project also provides everything you'll need on your zero trust journey.
+                    </p>
+                    <p>&nbsp;</p>
+                    <ul>
+                        <li><Link to="/docs/introduction/intro">Read the docs</Link></li>
+                        <li><Link to="/guides">Check out the guides</Link></li>
```

Comment:
I'm sure you know this, but just-in-case: /guides points to a "This will be where guides will be put" page